packages/app/src/app/store/sequences.js

```diff
@@ -172,15 +178,35 @@ export const forkSandbox = sequence('forkSandbox', [
   },
 ]);
 
+export const forkFrozenSandbox = sequence('forkFrozenSandbox', [
+  when(state`editor.currentSandbox.isFrozen`),
+  {
+    true: [
+      alertForkingFrozenSandbox,
+      {
+        confirmed: forkSandbox,
+        cancelled: [
+          set(props`message`, "Can't save a frozen sandbox"),
+          actions.callVSCodeCallbackError,
+          stopFrozenSandboxFromEdit,
+        ],
+      },
+    ],
+    false: [],
+  },
+]);
+
 export const ensureOwnedEditable = sequence('ensureOwnedEditable', [
-  when(
-    state`editor.currentSandbox.owned`,
-    state`editor.currentSandbox.isFrozen`,
-    (owned, frozen) => !owned || frozen
-  ),
+  when(state`editor.currentSandbox.owned`, owned => !owned),
```

Comment:
Can we reverse this one? It's a bit confusing for me to see a negation here 😅

Reply:
We can't reverse because of the old one checked if it was frozen too

What do you suggest?

Reply:
I'm thinking of this:

```
  when(state`editor.currentSandbox.owned`, owned => owned),
  {
  	true: [
      when(state`editor.currentSandbox.isFrozen`, isFrozen => isFrozen),
      {
        true: forkFrozenSandbox,
        false: [],
      },
    ],
    false: forkSandbox,
  },
```

Reply:
So flipping the true/false statements and setting `owned => owned`

Reply:
Done :)